exasol/analytics/query_handler/graph/stage/sql/dataset.py

```diff
@@ -13,6 +13,9 @@ class Dataset:
 
     The TableLike refers to a database table containing the actual data that
     can be used for instance in training or testing.
+
+    Q. A TableLike is basically a list of columns and a name. Why do we have
+    a separate list of columns here?
```

Comment:
AFAIK there was and is a separation of "ordinary" columns and "identifier columns" which may be handled specially in some cases? But the attribute `columns` could be superfluous in my understanding.

Reply:
Yeah, that's what I meant. The `identifier_columns` OK, I can understand. But just `columns` are inside the `table_like` and then also separately. Why?